apps/emergency-call/style/ice_contacts.css

```diff
@@ -17,8 +17,19 @@
   color: #FFF;
   line-height: 4.5rem;
   padding: 0 1.6rem 0 1.5rem;
+}
+
+.ice-contacts-bar > span.placeholder {
```

Comment:
Make this selector use the `ico-contacts-bar-arrow` id as describe above. Then there will be less chances of class name clash with shared CSS.